scripts/document.sh

```diff
@@ -53,7 +43,7 @@ find Sources/Mapbox{Core,}Navigation/ -name '*.swift' -exec \
 find Sources/Mapbox{Core,}Navigation/ -name '*.[hm]' -exec \
     perl -pi -e 's/([<"])MapboxCoreNavigation\b/$1MapboxNavigation/' {} \;
 
-jazzy \
+bundle exec jazzy \
```

Comment:
Do you have `bundle install` somewhere? I would even recommend running `bundle check || bundle install --path vendor/bundle` and cache gems:
```yaml
step-library:
    - &restore-gems-cache
        restore_cache:
            keys:
                - cache-v1-gems-{{ checksum "Gemfile.lock" }}

    - &save-gems-cache
        save_cache:
            key: cache-v1-gems-{{ checksum "Gemfile.lock" }}
            paths:
                - vendor/bundle
```

Reply:
I've added `bundle check || bundle install` to the script. As for caching, it's already implemented in job steps. I've renamed them to be more clear and universal. PTAL